.woodpecker/helm.yml

```diff
@@ -0,0 +1,36 @@
+clone:
+  git:
+    image: plugins/git:next
+
+pipeline:
+  release:
+    image: quay.io/helmpack/chart-releaser:v1.2.1
+    secrets:
+      - source: github_token
+        target: CR_TOKEN
+    commands:
+      - sed -i "s/<version>/${DRONE_TAG##v}/g" charts/woodpecker-agent/Chart.yaml
+      - sed -i "s/<version>/${DRONE_TAG##v}/g" charts/woodpecker-server/Chart.yaml
+      - cat charts/woodpecker-agent/Chart.yaml
+      - cat charts/woodpecker-server/Chart.yaml
+      - mkdir -p .cr-index
+      - cr package charts/woodpecker-server
+      - cr package charts/woodpecker-agent
+      - cr upload --owner woodpecker-ci --git-repo woodpecker-ci.github.io --release-name-template "helm-{{ .Name }}-{{ .Version }}"
+      - cr index --owner woodpecker-ci --git-repo woodpecker-ci.github.io --pages-branch main --charts-repo https://woodpecker-ci.github.io --push --release-name-template "helm-{{ .Name }}-{{ .Version }}"
+    when:
+      event: tag
+
+  lint:
```

Comment:
Shouldn't `lint` happen before `release`?

Reply:
You are right. I changed the order so that the lint can now abort the release and lint is execute on tag, but also on pr, o
push etc.